papaparse.js

```diff
@@ -878,7 +878,7 @@
 			_delimiterError = false;
 			if (!_config.delimiter)
 			{
-				var delimGuess = guessDelimiter(input, _config.newline, _config.skipEmptyLines);
+				var delimGuess = guessDelimiter(input, _config.newline);
```

Comment:
Why you removing it? Probably better to pass the config object directly. I prefer to not depend too much on global variables.

Reply:
Sure thing - I can add it back.

I removed it because I didn't think of _config as a global, but more of a property of the ParseHandle object.